test-end-to-end/test-descriptions/001_006_admin_can_cancel_content_download_test_description.md

```diff
@@ -0,0 +1,29 @@
+# Admin can cancel content download and download it again
+
+## Description:
+
+Content download can be canceled. After cancellation, the download stops and it can be
+downloaded again. (Android only).
```

Comment:
Should be Android and Desktop only. Even though we are not doing end-to-end automation on desktop at the moment.